Editor/Window/Components/ProfileSelector.cs

```diff
@@ -26,8 +25,9 @@ public ProfileSelector()
 
             LocalizeText();
 
-            _stateManager = EditorWindow.GetWindow<GameLiftPlugin>().StateManager;
+            _stateManager = EditorWindow.GetWindow<GameLiftPlugin>()._stateManager;
             _stateManager.OnProfileSelected += UpdateGUI;
+            _stateManager.OnBucketBootstrapped += UpdateGUI;
```

Comment:
Both these are meant to perform the same functionality. Please call this `OnProfileUpdated`and use just one callback in `StateManager` for the same.

Reply:
replaced